src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/todocomment/Example1.java

```diff
@@ -0,0 +1,18 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="TodoComment"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.todocomment;
+
+// xdoc section -- start
+public class Example1 {
+  int i;
+  public void test() {
+    i++; // TODO: do differently in future  // violation
+    i++; // todo: do differently in future
```

Comment:
please do examples to be the same and only comments are changing.
it is ok to merge examples together to archive required result.

Reply:
Done....
👍🏻